influxdb-mixin/dashboards/influxdb-cluster-overview.libsonnet

```diff
@@ -0,0 +1,1443 @@
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local getMatcher(cfg) = '%(influxdbSelector)s, influxdb_cluster=~"$influxdb_cluster"' % cfg;
+
+{
+  grafanaDashboards+:: {
+    'influxdb-cluster-overview.json':
+      dashboard.new(
+        'InfluxDB cluster overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+      .addLink(grafana.link.dashboards(
+        asDropdown=false,
+        title='Other InfluxDB dashboards',
+        includeVars=true,
+        keepTime=true,
+        tags=($._config.dashboardTags),
+      ))
+      .addTemplates(
+        [
+          template.datasource(
+            promDatasourceName,
+            'prometheus',
+            null,
+            label='Data Source',
+            refresh='load'
+          ),
+          template.new(
+            'job',
+            promDatasource,
+            'label_values(influxdb_uptime_seconds,job)',
+            label='Job',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.+',
+            sort=0
+          ),
+          template.new(
+            'cluster',
+            promDatasource,
+            'label_values(influxdb_uptime_seconds{%(multiclusterSelector)s}, cluster)' % $._config,
+            label='Cluster',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.*',
+            hide=if $._config.enableMultiCluster then '' else 'variable',
+            sort=0
+          ),
+          template.new(
+            'influxdb_cluster',
+            promDatasource,
+            'label_values(influxdb_uptime_seconds{%(influxdbSelector)s}, influxdb_cluster)' % $._config,
+            label='InfluxDB cluster',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='',
+            sort=0
+          ),
+          template.custom(
+            'k',
+            query='5,10,20,50',
+            current='5',
+            label='Top node count',
+            refresh='never',
+            includeAll=false,
+            multi=false,
+            allValues='',
+          ),
+        ]
+      )
+      .addPanels(
+        [
+          alertsPanel(getMatcher($._config)) { gridPos: { h: 8, w: 7, x: 0, y: 0 } },
+          serversPanel(getMatcher($._config)) { gridPos: { h: 8, w: 17, x: 7, y: 0 } },
+          goRow { gridPos: { h: 1, w: 24, x: 0, y: 8 } },
+          topInstancesByHeapMemoryUsagePanel(getMatcher($._config)) { gridPos: { h: 8, w: 12, x: 0, y: 9 } },
+          topInstancesByGCCPUUsagePanel(getMatcher($._config)) { gridPos: { h: 8, w: 12, x: 12, y: 9 } },
```

Comment:
Same comment as on the instance dashboard, I'd likely move this row and its panels to the very bottom of the dashboard